src/measure_theory/group/measurable_equiv.lean

```diff
@@ -160,4 +160,18 @@ def inv (G) [measurable_space G] [has_involutive_inv G] [has_measurable_inv G] :
 lemma symm_inv {G} [measurable_space G] [has_involutive_inv G] [has_measurable_inv G] :
   (inv G).symm = inv G := rfl
 
+/-- `equiv.div_right` as a `measurable_equiv`. -/
+@[to_additive /-" `equiv.sub_right` as a `measurable_equiv` "-/]
```

Comment:
Why are there comment signs in the to_additive docstring? Same thing several times below: has something changed in the way docstrings are used by to_additive?

Reply:
You're asking why this is written as `/-" foo "-/` instead of `"foo"`?
I believe that the Lean parser parses the former precisely as a string, so they can be used interchangeably.
I personally find the former nicer, since then the text is highlighted as a docstring.